### PR TITLE
Add api server properties to conference registration recipe

### PR DESCRIPTION
**Pull request by hlbraddock**

### CODE REVIEW

**Comment by hlbraddock** [comment]
@ryancarlson review por favor

File: conf_registration_api/templates/default/conf_registration_api_properties.xml.erb [hunk 1]
```diff
@@ -37,6 +37,8 @@
 
     <entry key="crsEmail"><%= node['crs-api']['crs-email']%></entry>
 
-    <entry key="mode">debug</entry>
+    <entry key="apiServerProtocol"><%= node['crs-api']['api-server-protocol']%></entry>
+    <entry key="apiServerPort"><%= node['crs-api']['api-server-port']%></entry>
+    <entry key="apiServerName"><%= node['crs-api']['api-server-name']%></entry>
```
Comment (ryancarlson) [inline conf_registration_api/templates/default/conf_registration_api_properties.xml.erb:42]: Do you know where to set this variable in the Stack Settings?

**Comment by ryancarlson** [comment]
Looks good!

File: conf_registration_api/templates/default/conf_registration_api_properties.xml.erb (hunk 1 above)
Comment (hlbraddock) [inline conf_registration_api/templates/default/conf_registration_api_properties.xml.erb:42]: Yes, and have set it there.

**Pull request merged**